.github/workflows/update_version.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+import datetime
+
+with open("versioning.yml") as version_file:
+    versioning_info = yaml.safe_load(version_file)
+
+current_version = versioning_info["lichess_bot_version"]
+
+utc_datetime = datetime.datetime.utcnow()
+new_version = f"{utc_datetime.year}.{utc_datetime.month}.{utc_datetime.day}"
+if current_version.startswith(new_version):
```

Comment:
If `new_version` is `2023.1.10` and the `current_version` is `2023.1.1`, this will return true. The initial `new_version` needs a dot at the end.
```python
new_version = f"{utc_datetime.year}.{utc_datetime.month}.{utc_datetime.day}."
```